mlrun/datastore/storeytargets.py

```diff
@@ -19,6 +19,7 @@
 import mlrun.model_monitoring.helpers
 from mlrun.datastore.base import DataStore
 
+from ..platforms.iguazio import parse_path
```

Comment:
Not sure you're supposed to import enterprise code into general mlrun code.

Reply:
targets.py does it

Reply:
Oh well then.